tensorboard/plugins/beholder/beholder_demo.py

```diff
@@ -25,19 +25,17 @@
 import sys
 
 import tensorflow as tf
-from tensorflow.examples.tutorials.mnist import input_data
-
-from tensorboard.plugins.beholder.beholder import Beholder
+import tensorflow.examples.tutorials.mnist as mnist
+import tensorboard.plugins.beholder as beholder
```

Comment:
Nice. Yeah, `from tensorboard.plugins.beholder.beholder` was kind of awkward - this is much more preferable.